src/foam/nanos/referral/GrantAppStoreCapabilityRuleAction.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.referral',
```

Comment:
Since this rule action itself doesn't explicitly refer to referralfee capability it can move to crunch/ (where the appstore is).